data/seo.ts

```diff
@@ -8,4 +8,7 @@ export const blogSeo ={
     metaogurl: " https://www.travala.com/",
     metaogimage: "app\favicon.ico",
     canonicallink:"https://www.travala.com/blog",
+    blogcardimage1:"Blog post thumbnail",
```

Comment:
For alt text please use the term alt somewhere
You may refer to what's Bhupendra has done.
 iconImageAlt1: "wallet",
    iconImageAlt2: "user",
    iconImageAlt3: "gift",